frontend/src/Pages/MapView.js

```diff
@@ -1,10 +1,30 @@
 import React from 'react';
 import Map from '../Map/Map';
+import cloudyday2 from '../Icon/AnimateIcon/cloudy-day-2.svg';
+import cloudy from '../Icon/AnimateIcon/cloudy.svg';
+import day from '../Icon/AnimateIcon/day.svg';
+import rainy1 from '../Icon/AnimateIcon/rainy-1.svg';
+import rainy6 from '../Icon/AnimateIcon/rainy-6.svg';
+
+let icons = [cloudyday2, cloudy, day, rainy1, rainy6, day];
+
+const iconChooser = (icons) => {
+  let i = 0;
+  let testArray = [];
+  while (i < 238) {
+    let randomNum = Math.floor(Math.random() * 6);
```

Comment:
Miért pont 6? Ha ezt megmondod, akkor rájössz, hogy nem is 6.

Reply:
A másik ettől függetlenül: minden 10^16 alkalommal el fog crashelni.

Reply:
Bocs, nem, 1-et nem ad vissza a Math.random(), szóval nem fog crashelni soha, csak ha a 6 már nem 6 lesz.

Reply:
átírtam icons.length-re (már nem 6)